src/main/scala/com/google/cloud/genomics/spark/examples/GenomicsConf.scala

```diff
@@ -58,13 +58,18 @@ class GenomicsConf(arguments: Seq[String]) extends ScallopConf(arguments) {
   }
 
   def getReferences = {
-    this.references().split(",").map(contig => {
-      val data = contig.split(":")
-      (data(0), (data(1).toLong, data(2).toLong))
-    }).toMap
+    Contig.parseContigsFromCommandLine(this.references())
   }
 }
 
 class PcaConf(arguments: Seq[String]) extends GenomicsConf(arguments) {
   val numPc = opt[Int](default = Some(2))
+  val allReferences = opt[Boolean]()
```

Comment:
This could use a bit of documentation about how it trumps --references if both are specified.

Also, in general we don't want all references for PCA, we want to leave out X and Y.  Perhaps --all-autosomes-references?  And then we trim X and Y out of the list returned by Contig.getContigsInVariantSet?

@doll does that line up with the plan for DataFlow PCA?